src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -350,13 +369,32 @@ def lvm_pv_type_label(device)
         format(_("PV of %s"), vg.basename)
       end
 
-      # Type label when the device belongs to a multidevice Btrfs filesystem
+      # Type label when the device holds a journal
       #
       # @param filesystem [Y2Storage::Filesystems::Base]
       # @return [String]
-      def btrfs_multidevice_type_label(filesystem)
-        # TRANSLATORS: %s is a device base name. E.g., sda1+
-        format(_("Part of Btrfs %s"), filesystem.blk_device_basename)
+      def journal_type_label(filesystem)
+        # TRANSLATORS: %{fs_name} is the filesystem name. E.g., Btrfs, Ext4, etc.
+        #              %{blk_device_name} is a device base name. E.g., sda1+
```

Comment:
"sda1+" ? We are not using + anymore.

Reply:
A simply copy paste from https://github.com/yast/yast-storage-ng/blob/41a8f597bf3b7a1d0432fefd80c0a21ceb01644a/src/lib/y2partitioner/widgets/blk_devices_table.rb#L357

There are a few occurrences in the file. I'll replace them by `sda1...`